packages/xo-server-test/src/backupNg/backupNg.spec.js

```diff
@@ -347,7 +347,7 @@ describe('backupNg', () => {
   test('execute three times a rolling snapshot with 2 as retention & revert to an old state', async () => {
     jest.setTimeout(6e4)
     await xo.createTempServer(config.servers.default)
-    const vmId = await xo.createTempVm({
+    let vm = await xo.createTempVm({
```

Comment:
This change does not improve the code, IMHO you should just save the id here.

Reply:
It replace  https://github.com/vatesfr/xen-orchestra/pull/4508/files#diff-59a6717e72428c99e66f7baf3e1f93deL381.